examples/qbv-example/.gitignore

```diff
@@ -0,0 +1,6 @@
+/bin
```

Comment:
This is fine, but I'm thinking a nice refactor (that should work) is to define these in a .gitignore in the ./examples directory, and then we can ideally remove this from this project and most/all of the other projects.